index.html

```diff
@@ -59,5 +57,249 @@ <h1 class="head-section">
       </div>
     </div>
   </section>
+  <section class="work" id="work">
+    <div class="card-work">
+      <div class="snapshot-portfolio">
+        <img src="snapshot-portfolio/tonic.png" alt="tonic">
+      </div>
+      <div class="left-block">
+        <h2 class="work-title">
+          Tonic
+        </h2>
+        <div class="frame">
+          <p class="client">
+            canopy
+          </p>
+          <img src="snapshot-portfolio/Counter.png" alt="counter">
+          <p class="role">
+            back end dev
+          </p>
+          <img src="snapshot-portfolio/Counter.png" alt="counter">
+          <p class="year">
+            2015
+          </p>
+        </div>
+        <div class="para">
```

Comment:
- [ ] As the previous code reviewer has mentioned. you need to stick to the Figma design.Kindly consider fixing the following problems.
- [ ] The items in the frame class though structured properly, lack the original design of the Template.
- [ ] The `see project` lack the `:active` class styling. In the figma design the active class should have `#2230D2` color.
- [ ] Kindly apply these changes in all your project cards.
your design
![qw](https://user-images.githubusercontent.com/96683843/212464935-a9684346-1cf2-45e2-883c-a23ef6e05464.jpg)

figma design
![qwe](https://user-images.githubusercontent.com/96683843/212464464-3005de4a-7b55-4d1a-be55-011277e795f2.jpg)